### ISSUE TITLE
[install] Does not install with yarn or npm

### ISSUE THREAD

**Issue by ryanhugh** [fomantic/Fomantic-UI#2461]
# Bug Report
```yarn add fomantic-ui``` fails. 

## Steps to reproduce

1. Create a new folder on your filesystem (outside of any JS package)
2. Run `yarn init`
3. Run `yarn add fomantic-ui`
4. It fails

## Expected result
I would expect it to install successfully.

## Actual result

This result in the terminal: 

```
yarn add v1.22.19
info No lockfile found.
[1/4] 🔍  Resolving packages...
warning fomantic-ui > gulp-concat-filenames > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
warning fomantic-ui > gulp-dedupe > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
warning fomantic-ui > gulp > glob-watcher > chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
warning fomantic-ui > gulp-concat-css > rework > css > source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
warning fomantic-ui > gulp-concat-css > rework > css > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning fomantic-ui > gulp-concat-css > rework > css > source-map-resolve > urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
warning fomantic-ui > gulp > glob-watcher > chokidar > fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.
warning fomantic-ui > gulp-concat-css > rework > css > source-map-resolve > resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
warning fomantic-ui > gulp-concat-css > rework > css > source-map-resolve > source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
warning fomantic-ui > gulp > gulp-cli > matchdep > micromatch > snapdragon > source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "fomantic-ui > @octokit/rest > @octokit/plugin-request-log@1.0.4" has unmet peer dependency "@octokit/core@>=3".
[4/4] 🔨  Building fresh packages...
[3/3] ⠈ fomantic-ui
[-/3] ⠈ waiting...
error /temp/node_modules/fomantic-ui: Command failed.
Exit signal: SIGINT
Command: gulp install
Arguments:
Directory: /temp/node_modules/fomantic-ui
Output:
[16:39:55] Using gulpfile /temp/node_modules/fomantic-ui/gulpfile.js
[16:39:55] Starting 'install'...
[16:39:55] Starting 'run setup'...
? Set-up Semantic UI (Use arrow keys)
❯ Automatic (Use default locations and all components)
  Express (Set components and output folder)
  Custom (Customize all src/dist values) [16:39:55] The following tasks did not complete: install, run setup

```

Some things are written to node_modules, but package.json is not updated. 

Using npm instead of yarn results in a similar error: 
```
4322 error command failed
4323 error signal SIGINT
4324 error command sh -c -- gulp install
4325 error [16:44:03] Using gulpfile /temp/node_modules/fomantic-ui/gulpfile.js
4325 error [16:44:03] Starting 'install'...
4325 error
[16:44:03] Starting 'run setup'...
4325 error ? Set-up Semantic UI (Use arrow keys)
4325 error ❯ Automatic (Use default locations and all components)
4325 error   Express (Set components and output folder)
4325 error   Custom (Customize all src/dist values) [16:44:03] The following tasks did not complete: install, run setup
4325 error [16:44:03] Did you forget to signal async completion?
4326 verbose exit 1

```

## Version

2.8.8

**Comment by lubber-de** [comment]
This should be fixed by #2272 
2.9.0 is supposed to be released next week, so in the meantime use the nightly build
```
yarn add fomantic-ui@nightly
```

**Comment by lubber-de** [comment]
Duplicate of #1911
